src/index.html

```diff
@@ -16,14 +16,14 @@ <h3>Recent Scores <span><button type="reset">Refresh</button></span></h3>
         <ul class="score-list">
```

Comment:
- [ ] OPTIONAL 
The _ReadMe_ for this project is **missing**. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, its contents need to reflect this project,  and may likely feature on your portfolio. Kindly ensure to add a ReadMe. You may use [this template](https://github.com/microverseinc/readme-template) to craft a professional ReadMe but remember to customize it to your project.